### DISCUSSION TITLE
Weird flash when changing view

### DISCUSSION THREAD

**Discussion by Mrcubix** [AvaloniaUI/Avalonia#8986] [Q&A] [answered]
I'm noticing this weird flash when changing views in my test app.
This happen on my main Windows 10 21H2 machine.

I also noticed on my windows 8.1 machine that avalonia print a notice about version 17763 being required for MAUI (2 for UWP i would assume).
The issue i'm talking about doesn't happen there.

https://user-images.githubusercontent.com/39861216/190901165-7f7768e0-f5d4-47b0-8b54-4c9586775cfa.mp4

This issue doesn't seem to happen every time.
After looking into it a bit more, it seem that the window become black for a couple of frames.

I'm using a simple navigation system most apps have so i don't think the issue come from that.

minimal demo: https://github.com/Mrcubix/Avalonia-Router-Test
Any ideas?

**Comment by timunie** [comment, marked as answer]
If you can provide a minimal demo App, we may have a look. Otherwise I can imagine that there is some I/O operation going on. 

One more question: does it only happen in debug mode or also in release mode?

**Reply by Mrcubix** [reply]
Will provide that once home (roughly 1h)

**Reply by timunie** [reply]
No rush. I will not be able to have a look today

**Reply by Mrcubix** [reply]
https://github.com/Mrcubix/Avalonia-Router-Test/tree/master/router_test here it is, almost the same as in the vid, which was already pretty minimal

**Reply by timunie** [reply]
It seems to be depended of the computer. In mine the flash is really minimal, but still annoying. It disappears if I set the background to the Window itself. I don't know if that's a possible solution for you. 
https://github.com/Mrcubix/Avalonia-Router-Test/pull/1

**Reply by Mrcubix** [reply]
I don't see anything different on my windows 8.1 laptop.
I will have to test it out on my main setup when i come back to my main home in a week.
Don't have any ways to test it on a windows 10 machine right now.

**Reply by Mrcubix** [reply]
Was able to test it and turns out that may have been the issue cause it doesn't happen on my main machine anymore, merged
